File: src/main/java/com/kingmang/lazurite/utils/Log.java
package com.kingmang.lazurite.utils;

import java.io.FileWriter;

public class Log {
    public static void append(String str) throws Exception {
        try (FileWriter writer = new FileWriter("log.txt", true)) {
            writer.write(str+"\n");
            writer.flush();
        }
    }
    public static void clear() throws Exception {
        try (FileWriter writer = new FileWriter("log.txt", false)) {

            writer.write("");
            writer.flush();
        }
    }
}

File: src/main/java/com/kingmang/lazurite/utils/Log.kt
package com.kingmang.lazurite.utils

import java.io.FileWriter

object Log {

    @JvmStatic
    fun append(str: String) {
        FileWriter("log.txt", true).use { writer ->
            writer.write(str)
            writer.appendLine()
            writer.flush()
        }
    }

    @JvmStatic
    fun clear() {
        FileWriter("log.txt", false).use { writer ->
            writer.write("")
            writer.flush()
        }
    }
}